<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog_UI</class>
 <widget class="QDialog" name="PreferencesDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="Layout33">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>184</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut">
       <string>Alt+C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;OK</string>
      </property>
      <property name="shortcut">
       <string>Alt+O</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="TabWidget2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>376</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>&amp;General</string>
    </attribute>
    <widget class="QCheckBox" name="restoreLastUsedSongCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Reopen last used song</string>
     </property>
     <property name="shortcut">
      <string>Alt+R</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="useLashCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Use lash</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sBcountOffset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>52</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>first step, adjust timing mismatch between controller/keyboard trigger latency and computed bpm</string>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLabel" name="bClabel_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Beat counter drift compensation in 1/10 ms   </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sBstartOffset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>52</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>second step, adjust offset between last controller/keybord trigger and the deferred sequencer startup </string>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Beat counter start offset in ms    </string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>511</width>
       <height>49</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Path to the Rubberband command-line utility</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="rubberbandLineEdit">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum number of bars</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sBmaxBars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>52</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>second step, adjust offset between last controller/keybord trigger and the deferred sequencer startup </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="restoreLastUsedPlaylistCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Reopen last used playlist</string>
     </property>
     <property name="shortcut">
      <string>Alt+R</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Audio &amp;System</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>10</y>
       <width>186</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Audio output details</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="driverInfoLbl">
        <property name="text">
         <string>###</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>336</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="driverComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="restartDriverBtn">
          <property name="text">
           <string>Restart output</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="1" column="1">
         <widget class="QSpinBox" name="bufferSizeSpinBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="sampleRateLbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Sample rate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="audioDeviceLbl">
          <property name="text">
           <string>Device</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="buffer_sizeLbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Buffer size</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_pAudioDeviceTxt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="sampleRateComboBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>44100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>88200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>96000</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="trackOutputComboBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Post-Fader</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pre-Fader</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="trackOutLbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Track output</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="connectDefaultsCheckBox">
          <property name="text">
           <string>Connect to &amp;Default Output Pair</string>
          </property>
          <property name="shortcut">
           <string>Alt+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="trackOutsCheckBox">
          <property name="text">
           <string>create per-instrument outputs</string>
          </property>
          <property name="shortcut">
           <string>Alt+D</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="1">
         <widget class="QSpinBox" name="maxVoicesTxt">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="metronomeVolumeSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TextLabel1_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Metronome volume</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel3_3">
          <property name="text">
           <string>Polyphony</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>&amp;Midi System</string>
    </attribute>
    <widget class="MidiTable" name="midiTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>141</y>
       <width>521</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>516</width>
       <height>135</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="textLabel1_5">
          <property name="text">
           <string>Midi driver</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_pMidiDriverComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="textLabel1_3_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="midiPortComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="midiPortChannelComboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="m_pIgnoreNoteOffCheckBox">
          <property name="text">
           <string>Ignore note-off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>&amp;Appearance</string>
    </attribute>
    <widget class="QLabel" name="textLabel3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>521</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;i&gt;After a font change the application must be restarted.&lt;/i&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>64</height>
      </rect>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="applicationFontLbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Application font</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1_4">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="styleComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="selectApplicationFontBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Select font</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>521</width>
       <height>66</height>
      </rect>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="selectMixerFontBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Select font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Meters falloff speed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mixerFalloffComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Slow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mixerFontLbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Mixer font</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MidiTable</class>
   <extends>QTableWidget</extends>
   <header location="global">../src/widgets/midiTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
